### ISSUE TITLE
Invalid Index when calling `key_state()` in v2

### ISSUE THREAD

**Issue by newbiemate** [spyoungtech/ahk#277]
### describe your issue

Something with `key_state()` broke between v1 and v2:

```
check = ahk.key_state("~")
```

This was working fine in v1. 

In v2, I got an `invalid index` error. The code didn't change between v1 -> v2, I only upgraded the ahk + ahk (python, v1.5.3) libraries.

### ahk.__version__

2.0.12

### AutoHotkey version

v2

### Code to reproduce the issue

```python
from ahk import AHK

ahk = AHK()
check = ahk.key_state("~")
```

### Traceback/Error message

```
Traceback (most recent call last):
  File "C:\test\ahk\testpy", line 4, in <module>
    check = ahk.key_state("~")
  File "C:\test\ahk\env\lib\site-packages\ahk\_sync\engine.py", line 1092, in key_state
    resp = self._transport.function_call('AHKKeyState', args, blocking=blocking)
  File "C:\test\ahk\env\lib\site-packages\ahk\_sync\transport.py", line 611, in function_call
    return self.send(request, engine=engine)
  File "C:\test\ahk\env\lib\site-packages\ahk\_sync\transport.py", line 832, in send
    return response.unpack()  # type: ignore
  File "C:\test\ahk\env\lib\site-packages\ahk\message.py", line 221, in unpack
    raise self._exception_type(s)
ahk.message.AHKExecutionException: Error occurred in Array.Prototype.__Item.Get (line 1671). The error message was: Invalid index.. Specifically: 2
```

**Comment by spyoungtech** [comment]
Thanks. Confirmed the bug and reproduced. When the `mode` argument is not given, this bug appears when using AHK v2.

I should have a fix out for this today.

**Comment by spyoungtech** [comment]
The fix for this is now released in version 1.5.4
